lua/ttt2/libraries/database.lua

```diff
@@ -393,310 +411,165 @@ end
 -- To be able to directly see message sending- and receive-structure between server and client
 -- They are always paired by either client-Send and server-Receive or server-Send and client-Receive
 
--- Client send and receive functions
-local clientSendFunctions = {}
+-- Client and server receive functions
 local clientReceiveFunctions = {}
-
--- Server send and receive functions
-local serverSendFunctions = {}
 local serverReceiveFunctions = {}
 
 ---
--- Send query for registered databases to server
--- @param table data contains only the messageIdentifier here
+-- An Identifier was received, checks if all sent identifiers were received and resets the identifiers
```

Comment:
```suggestion
-- An identifier was received, checks if all sent identifiers were received and resets the identifiers
```

But honestly I'm struggling to understand your sentence here. I don't get what you're trying to say

Reply:
This describes 2 functions. Overall description is, that an identifier was received, that is used to identify messages.
If the last identifier was received, this resets it.

Reply:
You don't have to erxplain it to me, but probably imporve the comment so that it reflects your thinking